src/views/DesignAssistantSurvey.js

```diff
@@ -304,10 +299,10 @@ class DesignAssistantSurvey extends Component {
   resetSurvey() {
     // do we need to delete submission object here?
     // because we would need to call the database here
-    this.state.model.clear();
+    // this.state.model.clear();
```

Comment:
why do we no longer want to wipe the model on reset?

Reply:
if we navigate by using `this.props.history.push({ pathname: '/' })` model will get remade anyway

unless i am misunderstanding what the reset button does